### PR TITLE
Update README.md

### LINKED ISSUE #39: Clarify if Python is needed in the README
Can you please clarify if Python is needed? Do you interface with Torch directly or Python must be installed?

### PR DESCRIPTION
**Pull request by juliohm**
Fix #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Torch.jl
 Sensible extensions for exposing torch in Julia.
 
-This package is aimed at providing the `Tensor` type, which offloads all computations over to [PyTorch](https://pytorch.org).
+This package is aimed at providing the `Tensor` type, which offloads all computations over to Torch.
```
Comment (DhairyaLGandhi) [inline README.md:4]: Best to keep the pytorch name there and disclaim that it isn't using python, since we do want to credit it appropriately.

**Comment by juliohm** [comment]
The statement is false. You are not sending the computation over to
pytorch. That is the whole point. If you want to say that this package is
the equivalent of the python wrapper, then do it with other words in
another sentence. This sentence is just confusing.

**Comment by ToucheSir** [comment]
If the intent is to eliminate any confusion or ambiguity, might I suggest the following:
```
... which offloads all computations to [LibTorch](https://pytorch.org/cppdocs/).
```
Mentioning just "Torch" is still ambiguous because readers may either
1. Infer "Torch" == "PyTorch" because they've only heard of the latter
2. Assume this is involved with Torch 7 (aka Lua Torch) somehow.

Since `libX` is a common convention for native libraries and the PyTorch docs already use the term "LibTorch" extensively, I'd argue this is the clearest way to explain that Torch.jl is wrapping/offloading to the PyTorch C++ API.

**Comment by juliohm** [comment]
Agree @ToucheSir , this is much clearer language and doesn't scare users who are only interested in Python-free solutions.

I will close this PR. Feel free to open another one.

**Pull request closed without merging**